server/routers/treesRouter.ts

```diff
@@ -100,7 +100,7 @@ router.get(
     }
     const result = await TreeModel.getByFilter(repo)(filter, options);
     res.send({
-      total: (startDate && endDate) || tag ? result.length : null,
+      total: result.length,
```

Comment:
@mayankbucha this is incorrect I think, the result length is just for the current page, so we need an extra SQL to `select count(*) ...` the same query maybe

Reply:
@dadiorchen we will have all the records in the result.length from the current query. So, total field will only need to have the number of records / results from the current query. 
Or should total return the number of results without limit added to the query?

Reply:
Yes, `total` is the number without the limit and offset, just the count(*)